tests/platfrom_specification_test.py

```diff
@@ -76,3 +80,34 @@ def test_if_validation_error_is_raised_for_incorrect_platform_schema(tmp_path):
     board_file.write_text(content)
     with pytest.raises(TwisterConfigurationException, match='Cannot create PlatformSpecification from yaml data'):
         PlatformSpecification.load_from_yaml(board_file)
+
+
+def test_if_discover_platform_revisions_returns_expected_platforms():
+    platform = PlatformSpecification('qemu_x86_tiny')
+    files = [
+        'qemu_x86_tiny_0.conf',  # valid
```

Comment:
Are those all expected (valid) formats for revisions?

Reply:
Now it works, but I do not know what is the valid format

Reply:
I found this. I believe this can be treated as background for valid revision formats https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#multiple-board-revisions

Reply:
and the comment in discover_platform_revison links to this function with its patterns https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/modules/extensions.cmake#L831